app/backend/src/clj/teet/cooperation/cooperation_db.clj

```diff
@@ -129,7 +130,37 @@
   (get-in (du/entity db application-id)
     [:cooperation.application/activity :db/id]))
 
+(defn applicaiton-3rd-party [db application-id]
+  (get-in (du/entity db application-id)
+    [:cooperation.3rd-party/_applications 0 :cooperation.3rd-party/name]))
+
 (defn response-project-id [db response-id]
   ;(def *args [db response-id])
   (get-in (du/entity db response-id)
           [:cooperation.application/_response 0 :cooperation.3rd-party/_applications 0 :cooperation.3rd-party/project :db/id]))
+
+(defn- is-date-after
```

Comment:
This should be in the date util, and check if any of the existing fns could be used.